crates/unftp-sbe-gcs/src/lib.rs

```diff
@@ -71,7 +72,7 @@ use bytes::Buf;
 use futures::{prelude::*, TryStreamExt};
 use hyper::{
     body::aggregate,
-    client::connect::{dns::GaiResolver, HttpConnector},
+    client::connect::{HttpConnector},
```

Comment:
Remove brackets, clippy will complain.

Reply:
Hmm, I run format on save which usually fixes this for me. Perhaps it didn't pick this one up.